src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowirePassTest.php

```diff
@@ -766,8 +769,13 @@ public function testWithNonExistingSetterAndAutowiring()
         (new AutowirePass())->process($container);
     }
 
+    /**
+     * @group legacy
+     */
     public function testExplicitMethodInjection()
     {
+        $this->expectDeprecation('Since symfony/dependency-injection 6.3: The "@required" annotation will be removed. Use the attribute "Symfony\Contracts\Service\Attribute\Required" instead.');
```

Comment:
```suggestion
        $this->expectDeprecation('Since symfony/dependency-injection 6.3: The "@required" annotation will be removed. Use the "Symfony\Contracts\Service\Attribute\Required" attribute instead.');
```